test/functional/ios/keyboard_tests.py

```diff
@@ -46,9 +48,14 @@ def test_hide_keyboard_presskey_strategy(self):
         self.assertFalse(el.is_displayed())
 
     def test_hide_keyboard_no_key_name(self):
+        if get_xcode_ver() == '11.3':
+            # TODO Needs to dismiss dictation dialog.
```

Comment:
The dialog is here. `autoAcceptAlerts: True` didn't work. 
<img src="https://user-images.githubusercontent.com/1549855/71639831-7fa43680-2cc1-11ea-9ddc-671cc4a81caa.png" width="320px">

Reply:
Thanks.
What the _Enable Dictation_ switch in _settings -> keyboards_ ?

My simulators were disabled it by default, but I assume it was enabled by default on your env. Then, the dialogue appears before typing texts. If it was enabled, what will happen when you disable it once?

Reply:
Thanks for your comments.

> What the Enable Dictation switch in settings -> keyboards ?
Unfortunately, I couldn't resolve by this in my local env since another dialog showed up. (Title: Enable 3rd party keyboard?)
But this behavior doesn't occur in ci env, so I'll resolve in my local env as another topic.